execution-engine/src/factory.rs

```diff
@@ -64,48 +63,23 @@ pub enum ExecutionStrategy {
 /// NB: wasmtime is only supported when feature=std is set at the moment,
 /// hence the branching around the body of this function.  When we get
 /// it compiled for SGX and TZ, then this will disappear.
+#[deprecated]
```

Comment:
Why `#[deprecated]`?  Is there a reason not to remove this function (and the other, below) immediately?

Reply:
This function is still in use in  the freestand-execution-engine.

Reply:
This function is still in use in the freestand-execution-engine.